src/client/components/card/crop-summary-card/crop-summary-card.component.js

```diff
@@ -1,53 +1,25 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import Card from '../card.component';
 import { CardTitle } from '../card-title.component';
-import { Status } from '../status-card/status-card.component';
+// import { Status } from '../status-card/status-card.component';
```

Comment:
Forgotten comment?

Reply:
intentionally left there to be fixed the future :)